include/CLI/App.hpp

```diff
@@ -1322,6 +1359,10 @@ class App {
         throw OptionNotFound(option_name);
     }
 
+    /// Shortcut bracket operator for getting results
+    const Option &operator[](std::string option_name) const { return *get_option(option_name); }
+    /// Shortcut bracket operator for getting results overload for const char *
+    const Option &operator[](const char *option_name) const { return *get_option(option_name); }
```

Comment:
Must have both, I guess? Disappointing that only `std::string` doesn't work.

Reply:
for some reason it was routing to a standard operator when you used app["--option"]  I didn't quite follow what was going on with the warning.  But it didn't seem to work unless I directly overloaded to const char *.  I might try a few more things and see if const std::string & works or something like that.

Reply:
I'm not shocked that it would require both, just saddened.